src/components/Backtester/reports/HistoricalReport.js

```diff
@@ -61,26 +87,63 @@ const HistoricalReport = (opts, results, backtestData, backtestOptions, t, setti
         execRunning={false}
       />
       {hasCandles && (
+        activeMarketObject && (
         <AutoSizer disableHeight style={{ height: 400 }}>
           {({ width, height = 400 }) => (
-            <BFXChart
-              indicators={indicators}
-              candles={candleArr}
-              trades={trades}
-              width={width}
-              height={height}
-              onAddIndicator={onAddIndicator}
-              onDeleteIndicator={onDeleteIndicator}
-              isSyncing={false}
-              candleLoadingThreshold={3} // we always get 1 candle when sub'ing
-              bgColor={chartColours.bgColor}
-              config={chartColours}
-              candleWidth={tf}
-              disableToolbar
-              showMarketLabel={false}
-            />
+            <div
+              style={full
+                ? FULL_SCREEN_STYLES
+                : { width, height }}
+              className='hfui-backtester__chart'
+            >
+              <Panel
+                label=''
+                removeable={false}
+                moveable={false}
+                extraIcons={[
+                  <span
+                    key='toggle-fullscreen'
+                    type='button'
+                    className='icon-move toggle-fullscreen'
+                    onClick={() => setFullScreenChart(!full)}
+                    title='Toggle Fullscreen'
+                  />,
+                ]}
+              >
+                <Chart
+                  market={activeMarketObject}
+                  theme={settingsTheme}
+                  layoutI='strategy-editor'
+                  indicators={chartIndicators}
+                  interval={interval}
+                  trades={trades}
+                  hideResolutions
+                />
+              </Panel>
+            </div>
           )}
         </AutoSizer>
+        )
+      // <AutoSizer disableHeight style={{ height: 400 }}>
+      //   {({ width, height = 400 }) => (
+      //     <BFXChart
+      //       indicators={indicators}
+      //       candles={candleArr}
+      //       trades={trades}
+      //       width={width}
+      //       height={height}
+      //       // onAddIndicator={onAddIndicator}
+      //       // onDeleteIndicator={onDeleteIndicator}
+      //       isSyncing={false}
+      //       candleLoadingThreshold={3} // we always get 1 candle when sub'ing
+      //       bgColor={chartColours.bgColor}
+      //       config={chartColours}
+      //       candleWidth={tf}
+      //       disableToolbar
+      //       showMarketLabel={false}
+      //     />
```

Comment:
keeping older chart code as commented for sometime to easily switch for debugging and compare features with tv-chart